docs: update creating vehicle and sensor model pages (#460)
autowarefoundation/autoware-documentation: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-model/index.md

# Creating vehicle and sensor models

## Overview

### Sensor Model

- **Purpose:** The sensor model includes the calibration (transformation) and launch files of the sensors used in the autonomous vehicle. This includes various sensors like LiDARs, cameras, radars, IMUs (Inertial Measurement Units), GPS units, etc.

> - **Importance:** Accurate sensor modeling is essential for perception tasks. True calibration values helps in understanding the environment by processing sensor data, such as detecting objects, estimating distances, and creating a 3D representation of the surroundings.

Author: done :+1:

> - **Usage:** The sensor model is used in Autoware for launching sensors, their pipeline and describing calibration values.

Author: done :+1:

- The sensor model (sensor kit) consists of three following packages:
  - `common_sensor_launch`
  - `<YOUR-VEHICLE-NAME>_sensor_kit_description`
  - `<YOUR-VEHICLE-NAME>_sensor_kit_launch`

> 

Author: done :+1:

Please follow the [creating sensor model](./creating-sensor-model) page
for creating your own individual sensor model.

> Here is the [sample_sensor_kit_launch](https://github.com/autowarefoundation/sample_sensor_kit_launch) package folder structure for autoware:

Author: done :+1:

```diff
sample_sensor_kit_launch/
├─ common_sensor_launch/
├─ sample_sensor_kit_description/
└─ sample_sensor_kit_launch/
```

### Vehicle Model

> - **Purpose:** The vehicle model includes individual vehicle specifications with dimensions, 3D model of vehicle (.fbx or .dae format), etc.

Author: done :+1:

- **Importance:** An accurate vehicle model is crucial for motion planning and control.
- **Usage:** The vehicle model is used in autoware for providing vehicle information for autoware, 3D model of vehicle, etc.

- The sensor model (sensor kit) consists of three following packages:
  - `<YOUR-VEHICLE-NAME>_vehicle_description`

>   - `<YOUR-VEHICLE-NAME>_vehicle_launch`

Reviewer: There is an important fix. This part should refer to the vehicle model, but it mentions the sensor kit.

>   - `<YOUR-VEHICLE-NAME>_vehicle_launch`

Author: done :+1:

> Please follow the [creating vehicle model](./creating-vehicle-model) page
> for creating your own individual vehicle model.
> 
> Here is the [sample_vehicle_launch](https://github.com/autowarefoundation/sample_vehicle_launch) package folder structure for autoware:
> 

Reviewer: ```suggestion
Please consult the [creating vehicle model](./creating-vehicle-model) page for creating your individual vehicle model.

As a reference, here is the folder structure for the [sample_vehicle_launch](https://github.com/autowarefoundation/sample_vehicle_launch) package in Autoware:

```

Author: done :+1:
